mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_WebChromeClient.java

```diff
@@ -208,4 +209,23 @@ public boolean onJsPrompt(final WebView view, final String url, final String mes
         return new IITC_JsDialogHelper(IITC_JsDialogHelper.PROMPT, view, url, message, defaultValue, result)
                 .shouldInterrupt();
     }
+
+    @Override
+    public boolean onCreateWindow(WebView view, boolean isDialog,
+        boolean isUserGesture, Message resultMsg) {
+
+        IITC_WebView newWebView = new IITC_WebView(mIitc);
+        view.addView(newWebView);
+        WebView.WebViewTransport transport = (WebView.WebViewTransport) resultMsg.obj;
+        transport.setWebView(newWebView);
+        resultMsg.sendToTarget();
+
+        return true;
+    }
+
+    public void onCloseWindow(WebView w){
+        if (mIitc.getWebView() != w)
+            mIitc.getWebView().removeView(w);
+        Log.d("onCloseWindow");
```

Comment:
Is that really useful?

Reply:
afair, It was, but it looks ok without atm

Reply:
nvm, It is useful, without, the popup can't close itself

Reply:
Actually I am asking about logging only.